src/App.tsx

```diff
@@ -1,61 +1,118 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import { TodoList } from './components/TodoList';
+import todosFromServer from './api/todos';
+import { Todo } from './Types/Todo';
+import { User } from './Types/User';
+
+const preparedTodos: Todo[] = todosFromServer.map(elem => {
+  return {
+    ...elem,
+    user: usersFromServer.find(user => user.id === elem.userId) || null,
+  };
+});
+
+function getLastIndex(todosList: Todo[]): number {
+  return Math.max(...todosList.map(currentTodo => currentTodo.id)) + 1;
+}
+
+function getUserById(selectedUserId: number): User {
+  return usersFromServer.find(user => user.id === selectedUserId) as User;
+}
 
 export const App = () => {
+  const [todosList, setTodosList] = useState<Todo[]>(preparedTodos);
+  const [title, setTitle] = useState('');
+  const [selectedUserId, setSelectedUserId] = useState(0);
+  const [isTitleFilled, setIsTitleFilled] = useState(true);
+  const [isUserSelected, setIsUserSelected] = useState(true);
+
+  const addTodo = (newTodo: Todo) => {
+    setTodosList(currentTodo => [...currentTodo, newTodo]);
+  };
+
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setIsTitleFilled(true);
+  };
+
+  const handleUserSelect = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setSelectedUserId(+event.target.value);
+    setIsUserSelected(true);
+  };
+
+  function reset() {
```

Comment:
```suggestion
  const reset = () => {
```
Create functions in one style